Author: gregor herrmann <gregoa@debian.org>
Bug: #482221
Description: The test depends on /etc/hosts or whatever resolves
'localhost'. Allow both 'localhost' and 'localhost.localdomain' as valid
values.

--- librpc-xml-perl-0.60.orig/t/40_server.t
+++ librpc-xml-perl-0.60/t/40_server.t
@@ -55,7 +55,7 @@
 $srv = RPC::XML::Server->new(no_default => 1,
                              host => 'localhost', port => $port);
 isa_ok($srv, 'RPC::XML::Server', '$srv<2>');
-is($srv->url, "http://localhost:$port/",
+like($srv->url, qr#http://localhost(.localdomain)?:$port/#,
    'RPC::XML::Server::url method (set)'); # This should be non-null this time
 # Test some of the simpler cases of add_method and get_method
 $res = $srv->add_method({ name      => 'perl.test.suite.test1',
